android/app/build.gradle

```diff
@@ -43,7 +43,7 @@ android {
     }
 
     defaultConfig {
-        applicationId "fr.neamar.cinetime"
+        applicationId "fr.neamar.cinetime.flutter"
```

Comment:
we can't change the package name on the Store, it has to remain fr.neamar.cinetime forever :(

Reply:
I know, you're right, it's just for now for testing internally.
It will be reverted to the current one once beta testing is over